content/knowledge/roadmap/timeline.en.yaml

```diff
@@ -246,48 +300,61 @@
   text: Classic community starts debating ETC monetary policy and emission schedule,
     as a way to align interests of platform users, miners, investors and
     developers
-- key: grothendieckTeam
+-
+  key: grothendieckTeam
   className: white
   link: https://ethereumclassic.github.io/blog/2016-12-12-TeamGrothendieck/
   icon: fas fa-user-ninja
   date: Dec 11 2016
   title: Grothendieck team
   text: A team of 7 full-time developers, committed by IOHK, starts working for
     Ethereum Classic
-- key: etcDiehardUpgrade
+-
+  key: diehard
   className: blue
   link: https://www.reddit.com/r/EthereumClassic/comments/5nt4qm/diehard_etc_protocol_upgrade_successful_nethash/
-  icon: fas fa-angle-double-up
-  date: Block 3000000
-  title: ETC Diehard Upgrade
-  text: ETC goes through a non-contentious network upgrade, resolving critical
-    issues such as difficulty bomb and replay attacks
-- key: etcDEVTeamAnnouncement
+  icon: fas fa-bomb
+  date: Jan 13 2017
+  title: Die Hard Upgrade
+  text: ETC goes through a non-contentious network upgrade, addressing replay attacks and delaying the difficulty bomb
+  -
```

Comment:
This line is indented incorrectly which is why the build is not working

You can see that the netlify bot didn't complete it's build:

![image](https://user-images.githubusercontent.com/1696942/78684165-0df30e80-78e8-11ea-958c-6bea60f4fd82.png)

Reply:
I've enabled gitpod for pull requests now so in future it will be much easier to tweak updates with a proper preview